src/client/utils/tracking/tracking.ts

```diff
@@ -116,14 +116,20 @@ export const adtraction = (
   }
 }
 
-export const trackStudentkortet = (memberId: string, amount: string) => {
-  const iframe = document.createElement('iframe')
-  iframe.src = `https://studentkortet.go2cloud.org/aff_l?offer_id=68&adv_sub=${memberId}&amount=${amount}`
-  iframe.scrolling = 'no'
-  iframe.frameBorder = '0'
-  iframe.width = '1'
-  iframe.height = '1'
-  document.body.appendChild(iframe)
+export const trackStudentkortet = (memberId: string) => {
+  const script = `!function(){var o=window.tdl=window.tdl||[];if(o.invoked)window.console&&console.error&&console.error("Tune snippet has been included more than once.");else{o.invoked=!0,o.methods=["init","identify","convert"],o.factory=function(n){return function(){var e=Array.prototype.slice.call(arguments);return e.unshift(n),o.push(e),o}};for(var e=0;e<o.methods.length;e++){var n=o.methods[e];o[n]=o.factory(n)}o.init=function(e){var n=document.createElement("script");n.type="text/javascript",n.async=!0,n.src="https://js.go2sdk.com/v2/tune.js";var t=document.getElementsByTagName("script")[0];t.parentNode.insertBefore(n,t),o.domain=e}}}();
+  tdl.init("https://aff.addreax.com/")
+  tdl.convert(
+  {
+  'adv_sub': '${memberId}'
+  }
+  )`
+  const scriptTag = window.document.createElement('script')
+  scriptTag.nonce = window.document.querySelector<HTMLScriptElement>(
+    '[nonce]',
+  )?.nonce
```

Comment:
Just a question though - is this the way you should add the `nonce` attribute? Asking since I'm not really sure myself.

Reply:
I think you have a point here ellen! This worked previously but a) it does not any more because of security updates to the browsers and b) we allow unsafe-inline nowadays because the tracking scripts are just too much a hassle otherwise :(

so to conclude, i think we don't need this line at all now